examples/hybrid/radiation_utilities.jl

```diff
@@ -242,6 +284,31 @@ function rrtmgp_model_callback!(integrator)
         end
     end
 
+    if !idealized_clouds && !(
+        rrtmgp_model.radiation_mode isa RRTMGPI.GrayRadiation ||
+        rrtmgp_model.radiation_mode isa RRTMGPI.ClearSkyRadiation
+    )
+        ᶜΔz = Fields.local_geometry_field(Y.c).∂x∂ξ.components.data.:9
+        ᶜlwp = RRTMGPI.array2field(
+            rrtmgp_model.center_cloud_liquid_water_path,
+            axes(Y.c),
+        )
+        ᶜiwp = RRTMGPI.array2field(
+            rrtmgp_model.center_cloud_ice_water_path,
+            axes(Y.c),
+        )
+        ᶜmask = RRTMGPI.array2field(
+            rrtmgp_model.center_cloud_boolean_mask,
+            axes(Y.c),
+        )
+        # multiply by 1000 to convert from kg/m^2 to g/m^2
+        @. ᶜlwp = 1000 * Y.c.ρ * TD.liquid_specific_humidity(params, ᶜts) * ᶜΔz
+        @. ᶜiwp = 1000 * Y.c.ρ * TD.ice_specific_humidity(params, ᶜts) * ᶜΔz
+        @. ᶜmask =
```

Comment:
What is this condition? We should probably add this as a thermo function. We have `has_condensate`, but this looks like a different condition?

Reply:
Oh! I made this up. We can use `has_condensate`, thanks! We may need a threshold at some point. I will let you know.